### PR TITLE
add select states

### PR DESCRIPTION
**Pull request by jeremy-babylonlabs**

### CODE REVIEW

File: src/components/Form/Select.stories.tsx [hunk 1]
```diff
@@ -85,3 +85,16 @@ export const CustomSelectedDisplay: Story = {
...
+export const WithError: Story = {
+  args: {
+    options: [
+      { value: "active", label: "Active" },
+      { value: "inactive", label: "Inactive" },
+      { value: "pending", label: "Pending" },
+    ],
+    placeholder: "Select status",
+    state: "error",
+    stateText: "This field is required",
```
Comment (totraev) [inline src/components/Form/Select.stories.tsx:98]: lets rename it to `hint`

File: src/components/Form/Select.tsx [hunk 2]
```diff
@@ -107,10 +113,10 @@ export const Select = forwardRef<HTMLDivElement, SelectProps>(
     }, [isOpen, disabled, setIsOpen]);
 
     return (
-      <>
+      <div className={twJoin("bbn-select-container", wrapperClassName)}>
```
Comment (totraev) [inline src/components/Form/Select.tsx:116]: Why do you need a wrapper here?

File: src/components/Form/Select.tsx [hunk 3]
```diff
@@ -142,7 +148,10 @@ export const Select = forwardRef<HTMLDivElement, SelectProps>(
             </div>
           ))}
         </Popover>
-      </>
+        {stateText && (
+          <span className={twJoin("bbn-select-state-text", `bbn-select-state-text-${state}`)}>{stateText}</span>
+        )}
+      </div>
```
Comment (totraev) [inline src/components/Form/Select.tsx:154]: Okay, got it. Lets remove `stateText` or `hint` from here. We'll create a separate component to wrap any form element. It'll add a label, hint and error

**Review by totraev — APPROVED** [review]

**Comment by jeremy-babylonlabs** [comment]
closing in favor or #59

**Pull request closed without merging**